test/sonatype-nexus3.test.ts

```diff
@@ -105,3 +105,45 @@ function overrideStackWithContextDomainName(app: cdk.App, stack: cdk.Stack,
   });
   return { app, stack };
 }
+
+describe('Nexus OSS stack and instanceType newVpc', () => {
+  let app: cdk.App;
+  let stack: cdk.Stack;
+  let previous: (scope: cdk.Construct, options: cdk.GetContextValueOptions) => cdk.GetContextValueResult;
+
+  afterAll(() => {
+    mock.restoreContextProvider(previous);
+  });
+
+  beforeEach(() => {
+    ({ app, stack } = overrideStackWithContextNewVpcInstanceType(app, stack, 'example.com','m5.xlarge', true));
+  });
+
+  test('Nexus Stack have new vpc and custom instanceType', () => {
+    expect(stack).toHaveResourceLike('AWS::CloudFormation::Stack', {
```

Comment:
As a unit test, this expected assertions will check if a new VPC resource is created and the managed node group of EKS using given instance type, for example `m5.xlarge` in this case. 😃

Reply:
no problem 😊  😊  😊 , will add it